recipes/r-x13binary/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set version = '1.1.39-2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-x13binary
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/x13binary_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/x13binary/x13binary_{{ version }}.tar.gz
+  sha256: b78383a956061405791439f6991670d2a53626add783cb1f4a391bbbbdf2c8c5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [osx]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('x13binary')"           # [not win]
+    - "\"%R%\" -e \"library('x13binary')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=x13binary
+  license: GPL-2.0-or-later
+  summary: The US Census Bureau provides a seasonal adjustment program now called 'X-13ARIMA-SEATS'
+    building on both earlier programs called X-11 and X-12 as well as the SEATS program
+    by the Bank of Spain. The US Census Bureau offers both source and binary versions
+    -- which this package integrates for use by other R packages.
+  license_family: OTHER
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - rluria14
+    - ndmaxar
+    - RyanHackney
```

Please add the conda-forge/r group:
```suggestion
    - RyanHackney
    - conda-forge/r
```